### PR TITLE
Rust SIMD

### PR DESCRIPTION
**Pull request by darleybarreto**
Inspired by the Julia SIMD version, I've implemented a simple SIMD Rust counterpart. I also tried to perform a parallel version of it, but my Rust skills aren't quite there yet, so I couldn't figure out a nice way of doing it. There was a merge conflict on the `.gitignore`, so I solved and rebased.

### CODE REVIEW

File: rust/nbabel-nbody/src/simd.rs [hunk 1]
```diff
@@ -0,0 +1,127 @@
+//! Based on https://github.com/rust-lang/packed_simd/blob/master/examples/nbody/src/simd.rs
+use std::path::Path;
+use std::{fs, mem};
+
+use packed_simd::*;
```
Comment (ashwinvis) [inline rust/nbabel-nbody/src/simd.rs:5]: Can we use explicit import here? It seems that only `f64x4` is required.
```suggestion
use packed_simd::f64x4;
```

Comment (darleybarreto) [reply rust/nbabel-nbody/src/simd.rs:5]: You are right, I thought `f64x64` would need more traits imported from the root package.

**Comment by paugier** [comment]
I don't understand why py/bench_pypy4.py is modified by this branch and why there is commit 04f87ff88811a08c5346a8c0916cc4d9a01e4808. Can you please fix that so that I can merge?

**Comment by darleybarreto** [comment]
I'm not sure what's happening there, actually. I started developing the SIMD version a while ago, then I squased the commits, after that I rebased, there was a conflict in `.gitignore`, it may be due to that? One option is to rebase before `04f87ff`, add my changes and PR.

**Comment by darleybarreto** [comment]
I think now we can merge this.

**Comment by paugier** [comment]
Thanks! I will try your new implementation as soon as I can.

**Comment by darleybarreto** [comment]
Oh gosh, I just realized that I forgot to change from implicit to explicit import... So, when you run the benchmark, could you kindly replace this [line](https://github.com/paugier/nbabel/blob/master/rust/nbabel-nbody/src/simd.rs#L5) with the following?
```rust
use packed_simd::f64x4;
```
So another commit from me won't be needed.

**Pull request merged**